react/src/js/react/posts/PostsStore.js

```diff
@@ -0,0 +1,109 @@
+import Reflux from "reflux";
```

Comment:
It may make more sense to move this store into the discovery folder. This is meant just for the Discover page, and we already are going to have at least 2 other stores that are also meant for posts so it's kind of confusing to have this one be here